Npm init

pedroespindula opened the pull request:
**Feature/Bug description:** Prepares repository for npm app development.

**Solution:** Add package.json, .gitignore and main.js.

**TODO/FIXME:** n/a

Commit:
Add .gitignore

Commit:
Add package.json

Commit:
Create main.js

brobsc commented on .gitignore:
```diff
@@ -0,0 +1,17 @@
+yarn-debug.log*
+yarn-error.log*
+
+# Lock files
```
Lock files should be comitted.

JuliePessoa replied:
why?

pedroespindula replied:
**Solved**. When you use more than one kind of lock file, it may lead to conflict. The chosen one to be used for this repository was package-lock.json (npm), and the ignored one was yarn.lock (yarn). For more info see https://stackoverflow.com/questions/44552348/should-i-commit-yarn-lock-and-package-lock-json-files

JuliePessoa replied:
ok. so commit the lock file.

Commit:
Remove package-lock.json from .gitignore

A lock file must be commited but not both of them. When both are commited, it may lead to a conflict.
